tests/end-to-end/sandbox/_files/BeforeAndAfterClassMethodCallCountTest.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\TestFixture;
+
+use function file_get_contents;
+use function file_put_contents;
+use function intval;
+use PHPUnit\Framework\Attributes\RunClassInSeparateProcess;
+use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
+use PHPUnit\Framework\TestCase;
+
+//#[RunTestsInSeparateProcesses]
+#[RunClassInSeparateProcess]
+final class BeforeAndAfterClassMethodCallCountTest extends TestCase
```

Comment:
Simillar test would be good for a class with some tests run in a separate process and some not.